spec/javascripts/unit/admin/enterprises/services/enterprise_shipping_methods_spec.js.coffee

```diff
@@ -1,44 +0,0 @@
-describe "EnterpriseShippingMethods service", ->
```

Comment:
Did you intend to delete the service as well? `app/assets/javascripts/admin/enterprises/services/enterprise_shipping_methods.js.coffee`

Reply:
I just pushed smoothing out all the broken tests. Could you take a look please?